ipfs-pinning-service.yaml

```diff
@@ -45,6 +45,25 @@ response. The old pin object is deleted automatically.
 
 Pin object can be removed via `DELETE /pins/{cid-of-pin-object}`
 
+# Custom metadata
+
+Pinning Services are encouraged to add support for additional features by
+leveraging optional `meta` attributes:
+
+- `PinStatus.meta[progress]` –  % progress of ongoing pinning operation
+
+- `PinStatus.meta[receivers] = ['multiaddr1','multiaddr2']` list of peers to connect to to speed up transfer of pinned data
```

Comment:
Not necessarily related to the spec, but I'd really like to emphasize to current pinning services that this is really useful and they should implement it if it's not a huge ask. If many of them are unable to implement it in a reasonable time frame then we should be aware of that when dealing with user issues.

Reply:
@aschmahmann I agree this is pretty important feature, I worry if we leave it in `meta` it may be harder for services to implement it. Should we promote this to a required (could be an empty array) top level field(s)?
Would it be ok to rename it to `providers` everywhere?